File: src/net/liplum/DstItems.java
package net.liplum;

import arc.graphics.Color;
import mindustry.ctype.ContentList;
import mindustry.type.Item;

public class DstItems implements ContentList {
    public static Item thulium;
    public static Item thuliumBar;

    @Override
    public void load() {
        thulium = new Item("thulium", Color.valueOf("#f7ae08")) {{
            hardness = 0;
            radioactivity = 0.25f;
            cost = 3.5f;
        }};
        thuliumBar = new Item("thulium-bar", Color.valueOf("#ce7508")) {{
            radioactivity = 0.25f;
        }};
    }
}

File: src/net/liplum/DstItems.kt
package net.liplum

import arc.graphics.Color
import mindustry.type.Item

object DstItems : ContentList {
    @JvmStatic lateinit var thulium: Item
    @JvmStatic lateinit var thuliumBar: Item
    override fun load() {
        thulium = Item("thulium", Color.valueOf("#f7ae08")).apply {
            hardness = 0
            radioactivity = 0.25f
            cost = 3.5f
        }
        thuliumBar = Item("thulium-bar", Color.valueOf("#ce7508")).apply {
            radioactivity = 0.25f
        }
    }
}